src/lifetracker/logic/StorageAdapter.java

```diff
@@ -0,0 +1,13 @@
+package lifetracker.logic;
+
+import java.io.IOException;
+
+import lifetracker.calendar.CalendarList;
+import lifetracker.storage.Storage;
+
+public interface StorageAdapter {
```

Comment:
As mentioned, remove the interface, since StroageAdapter is not used by other classes outside of logic. Just name the implementation as StorageAdapter